packages/@react-native-windows/telemetry/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "@rnw-scripts/ts-config",
+  "compilerOptions": {
+    // Change the default to allow ES2019.String "trimEnd"
+    "lib": [
+      "DOM",
+      "ES6",
+      "DOM.Iterable",
+      "ES2019.String"
+    ],
+
+    // Not clean
+    "strictFunctionTypes": false,
+  },
```

Comment:
Nit for the version of this that got checked into master, we shouldn't need the highlighted chunk, as these bits were all specific to the CLI. The top was telated to existing usages of some string APIs we wanted to allow, and the bottom was a warning we needed to suppress for a specific package.